### PR TITLE
UT for checking exception, when filter passes data with single date

### PR DESCRIPTION
**Pull request by zBritva**

### CODE REVIEW

File: test/visualTest.ts [hunk 1]
```diff
@@ -203,6 +204,33 @@ namespace powerbi.extensibility.visual.test {
                     });
                 });
             });
+
+            describe("selection with single date", () => {
+                beforeEach(() => {
+                    visualBuilder = new PulseChartBuilder(400, 150);
```
Comment (ignatvilesov) [inline test/visualTest.ts]: Why do you create a visual builder again?

File: test/visualTest.ts [hunk 2]
```diff
@@ -203,6 +204,33 @@ namespace powerbi.extensibility.visual.test {
...
+                    dataViewSingleDate = defaultDataViewBuilder.getDataViewWithSingleDate();
+                });
+
+                it("select click", (done) => {
+                    visualBuilder.updateRenderTimeout(dataView, () => {
+                        //apply filtered date
+                        try {
+                            visualBuilder.updateRenderTimeout(dataViewSingleDate, () => {
+                                debugger;
```
Comment (ignatvilesov) [inline test/visualTest.ts]: Please remove the ```debugger``` statement.

File: test/visualTest.ts [hunk 3]
```diff
@@ -203,6 +204,33 @@ namespace powerbi.extensibility.visual.test {
...
+
+                                const clickPoint: JQuery = visualBuilder.mainElement.first();
+                                clickPoint.click();
+                                setTimeout(() => {
```
Comment (ignatvilesov) [inline test/visualTest.ts]: Do we really need to use this timeout?

File: test/visualTest.ts [hunk 4]
```diff
@@ -203,6 +204,33 @@ namespace powerbi.extensibility.visual.test {
...
+                                    done();
+                                }, DefaultTimeout);
+                            }, DefaultTimeout);
+                        }
+                        catch(exception){
+                            expect(exception).not.toBe("Cannot read property \'0\' of undefined");
```
Comment (ignatvilesov) [inline test/visualTest.ts]: I suppose that you can rewrite this test case using
```
expect(() => {
    // Code
}).not.toThrow(); // not.toThrowError()
```

File: test/visualTest.ts (hunk 1 above)
Comment (zBritva) [inline test/visualTest.ts]: Removed

File: test/visualTest.ts (hunk 2 above)
Comment (zBritva) [inline test/visualTest.ts]: Removed

File: test/visualTest.ts (hunk 4 above)
Comment (zBritva) [inline test/visualTest.ts]: Done

File: test/visualTest.ts (hunk 3 above)
Comment (zBritva) [inline test/visualTest.ts]: removed

File: test/visualTest.ts [hunk 5]
```diff
@@ -39,7 +39,8 @@ namespace powerbi.extensibility.visual.test {
     describe("PulseChartTests", () => {
         let visualBuilder: PulseChartBuilder,
             defaultDataViewBuilder: PulseChartData,
-            dataView: DataView;
+            dataView: DataView,
+            dataViewSingleDate: DataView;
```
Comment (ignatvilesov) [inline test/visualTest.ts]: Please move ```dataViewSingleDate``` to the ```describe("selection with single date", () => {``` below.

**Comment by zBritva** [comment]
Changes were moved to https://github.com/Microsoft/powerbi-visuals-pulsechart/pull/4

**Pull request closed without merging**